macros/generic_tests/at_least_one.sql

```diff
@@ -4,9 +4,17 @@
 
 {% macro default__test_at_least_one(model, column_name, group_by_columns) %}
 
-{% if group_by_columns|length() > 0 %}
-  {% set select_gb_cols = group_by_columns|join(' ,') + ', ' %}
-  {% set groupby_gb_cols = 'group by ' + group_by_columns|join(',') %}
+{#- Ensure that column_name is not in the group by list or the database will throw an ambiguous column error #}
+{%- set group_by_cols_filtered = [] %}
+{% for column in group_by_columns %}
+  {%- if column|upper != column_name|upper %}
+      {%- do group_by_cols_filtered.append(column) %}
+  {% endif -%}
+{% endfor %}
```

Comment:
I don't think this is necessary - including the name of the column you're testing in your group by columns as well is a misconfiguration that we shouldn't allow. 

Arguably, it's nice that we fix it for you instead of penalising you, but if we do that then we should make the change across all of the macros that have this behaviour and it should be in a different PR.